src/forklift/core.py

```diff
@@ -134,9 +131,14 @@ def update(crate, validate_crate):
                                 continue
 
                             cursor.insertRow(row)
+        else:
+            log.debug('No changes found.')
+
+            if changes.has_dups:
+                change_status = (Crate.WARNING, 'Duplicate features detected!')
 
         if changes.has_dups:
```

Comment:
Should this if block be moved to 133 within the `if changes.has_changes():` block?

Reply:
yuuup